### PR TITLE
Move client bats tests to a stand alone file

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Review by ekohl — COMMENTED** [review]
Looks like I forgot to submit my review last week.

File: bats/fb-content-client.bats [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+load foreman_helper
+load fixtures/content
+
+@test "install subscription manager" {
+  tPackageExists subscription-manager || tPackageInstall subscription-manager
+}
+
+@test "disable puppet agent to prevent checkin from registering host to another org" {
+  systemctl is-active puppet || skip "Puppet is not active"
+  systemctl stop puppet
```
[the remark this answers is not in the document]
Comment (ehelms) [reply bats/fb-content-client.bats:16]: That existed before. What are you asking me to change here?

File: bats/fb-content-client.bats [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  if [ -e "/etc/rhsm/ca/candlepin-local.pem" ]; then
+    rpm -e `rpm -qf /etc/rhsm/ca/candlepin-local.pem`
+  fi
+
+  cleanSubscriptionManager
+
+  run yum erase -y 'katello-ca-consumer-*'
+  echo "rc=${status}"
+  echo "${output}"
+  run rpm -Uvh http://localhost/pub/katello-ca-consumer-latest.noarch.rpm
```
[the remark this answers is not in the document]
Comment (ehelms) [reply bats/fb-content-client.bats:33]: Yep. I am simply moving the tests into a single file together. They run the same as they did before. My intent is not to change the tests but rather to group them into a dedicated test file.

**Comment by evgeni** [comment]
So the idea here is to move the client-specific tests out to an own file, so that in the next step we can run the tests on a separate client box, right? Generally, I like that.

I dislike the fact that this now makes `fb-content-client` implicitly depend on `fb-content-katello` (and yes I know, it's no different with `fb-proxy`), but at the same time, this is what bats gives/allows us, so not blocking on this.

Can we give the files sane names please? `fb-katello-content`, `fb-katello-proxy`, `fb-katello-client`? (And we should drop that `fb` prefix at some point, but not in this PR).

These files could also need some cleanup, but not today.

**Comment by ehelms** [comment]
You wanting me to do a rename of the files in this PR?

**Comment by evgeni** [comment]
at least for content-katello and content-client I'd prefer to, yes.

**Review by evgeni — APPROVED** [review]
Thanks!

Can you also open a PR for downstream?

**Pull request merged**